src/webportal/src/app/vc/vc.component.js

```diff
@@ -54,6 +60,8 @@ const loadData = (specifiedVc) => {
           {type: 'natural', targets: [0, 1, 2, 3, 4, 5, 6]},
         ],
       }).api();
+      resizeContentWrapper();
+      $('.state-vc .tips').html('Click To Change Status');
```

Comment:
If the `.tips` only contains such simple text, `title` attribute may be better. Bootstrap also provides [tooltips component](https://getbootstrap.com/docs/3.3/javascript/#tooltips)